src/sharedServices/dynamoDB/dynamoDBUpdate.spec.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  DynamoDBClient,
+  UpdateItemCommand,
+  UpdateItemOutput
+} from '@aws-sdk/client-dynamodb'
+import { mockClient } from 'aws-sdk-client-mock'
+import { updateQueryByZendeskId } from './dynamoDBUpdate'
+
+const dynamoMock = mockClient(DynamoDBClient)
+
+describe('dynamoDBUpdate', () => {
+  beforeEach(() => {
+    dynamoMock.reset()
+  })
+
+  test('Updates a request query in database', async () => {
+    const mockUpdatedDbContents = {
+      Attributes: {
+        requestInfo: {
+          M: {
+            recipientName: { S: 'test' },
+            dateTo: { S: '2022-09-06' },
+            identifierType: { S: 'eventId' },
+            dateFrom: { S: '2022-09-06' },
+            zendeskId: { S: '12' },
+            eventIds: { L: [{ S: '234gh24' }, { S: '98h98bc' }] },
+            piiTypes: { L: [{ S: 'passport_number' }] },
+            recipientEmail: { S: 'test@test.gov.uk' },
+            requesterEmail: { S: 'test@test.gov.uk' },
+            requesterName: { S: 'test' }
+          }
+        },
+        zendeskId: { S: '12' },
+        athenaQueryId: { S: 'testAthenaId' }
+      }
+    }
+
+    dynamoMock
+      .on(UpdateItemCommand)
+      .resolves(mockUpdatedDbContents as UpdateItemOutput)
+
+    const result = await updateQueryByZendeskId(
+      '12',
+      'athenaQueryId',
+      'testAthenaId'
+    )
+    expect(result).toEqual({
```

Comment:
Similarly to above, you can check that the right commands were passed using `toHaveReceivedCommandWith`

Reply:
Done